sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -37,7 +37,7 @@ AppConfiguration to use Netty HTTP client.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-http-netty</artifactId>
-    <version>1.2.0</version>
+    <version>1.3.0-beta.1</version>
```

Comment:
Should revert all the README changes, this hasn't been released yet and we don't want to accidentally release all the READMEs with non-existent versions.

Reply:
If we're holding off on updating the versions then just run the script again with the --sr option to skip the readme updates. Though, with that being said, were the readme versions for the artifacts we're releasing correct before the release of those respective artifacts.